include/libtorrent/bt_peer_connection.hpp

```diff
@@ -272,6 +272,7 @@ namespace libtorrent
 #endif
 
 	private:
+        void write_dht_port();
```

Comment:
sorry, I didn't notice this the first time, but this indentation isn't right either, it should be tabs.